str_analysis/call_non_ref_pathogenic_motifs.py

```diff
@@ -953,6 +1025,29 @@ def process_locus(locus_id, args, normalize_to_coverage=NORMALIZE_TO_COVERAGE, m
 
     print(f"Final call: {final_call}")
 
+    # Process STRling results
+    if strling_genotype_df is not None:
+        strling_records = parse_strling_info_for_locus(strling_genotype_df, locus_chrom, locus_start_0based, locus_end)
+        for i in 0, 1:
+            record = {}
+            if i < len(strling_records):
+                record = strling_records[i]
+            elif i > 0:
+                break
+
+            motif_number = i + 1
+            locus_results_json.update({
+                f"strling_motif{motif_number}_repeat_unit": record.get("repeatunit"),
+                f"strling_motif{motif_number}_start_1based": record.get("left"),
+                f"strling_motif{motif_number}_end_1based": record.get("right"),
```

Comment:
these two fields are described in the STRling docs (https://strling.readthedocs.io/en/latest/outputs.html) but not in this example https://github.com/broadinstitute/str-analysis/blob/main/str_analysis/utils/strling_info_for_locus.py#L24